env.default

```diff
@@ -75,3 +75,7 @@ USE_CLOUDINARY=False
 CLOUDINARY_CLOUD_NAME=""
 CLOUDINARY_API_KEY=""
 CLOUDINARY_API_SECRET=""
+
+# Selenium testing
+FIREFOX_BINARY=
```

Comment:
If you don't add quotes, dotenv inside pipenv will make a mess and think that CHROME_BINARY is the value of FIREFOX_BINARY. FUN STUFF 😞

Reply:
... what, that's crazy. Empty quotes it is =_=

Reply:
updated